package.json

```diff
@@ -11,10 +11,11 @@
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
     "react-relay": "^10.1.1",
+    "react-router-dom": "^5.2.0",
     "relay": "^0.8.0-1"
   },
   "scripts": {
-    "relay": "relay-compiler --src ./src --schema schema.graphql --extensions ts tsx --language typescript",
+    "relay": "relay-compiler --src ./src --schema src/graphql/schema/fakeArtsy.graphql  --extensions ts tsx --language typescript",
```

Comment:
This is actually a fix for my previously merged PR, #3. I missed the fact that the relay task no longer worked because it was pointed at a non-existent schema.